server/resources/art/artModel.js

```diff
@@ -0,0 +1,11 @@
+var Sequelize = require('sequelize');
+var db = require('../../db/db.js');
+
+var Art = db.define('Art', {
+  type: {
```

Comment:
all of our model should have auto-id's, but for this preliminary pull not having one is fine